packages/ui/BodyText/BodyText.js

```diff
@@ -35,6 +35,16 @@ const BodyText = kind({
 		 */
 		centered: PropTypes.bool,
```

Comment:
If we support centered, should we support right-align as well?

Reply:
It's possible, but that may be a deeper consideration for BodyText than the intent of this ticket.

To do it the way I'd want, which would be to mirror marquee's `alignment` prop, we'd need to deprecate `centered` and include some handling/conversion for it.

Reply:
Sounds good. That could be another issue that's out of this scope.